targets/switchboard/artillery-scripts/send-message.yaml

```diff
@@ -0,0 +1,106 @@
+config:
+  environments:
+    staging:
+      target: "https://staging.switchboard.assemble.live"
+      phases:
+        - name: Spike - Spoke Autosending @ 100
+          duration: 180
+          arrivalRate: 100
+        - name: Spike - Spoke Autosending @ 200
+          duration: 180
+          arrivalRate: 200
+        - name: Spike - Spoke Autosending @ 300
+          duration: 180
+          arrivalRate: 300
+        - name: Spike - Spoke Autosending @ 400
+          duration: 180
+          arrivalRate: 400
+        - name: Spike - Spoke Autosending @ 500
+          duration: 180
+          arrivalRate: 500
+  plugins:
+    metrics-by-endpoint: {}
+  processor: "../processors/sendMessageProcessor.js"
+
+scenarios:
+  - name: "Sending Bandwidth messages"
+    flow:
+      # Execute sendMessage mutation
+      - post:
+          url: "/sms/graphql"
+          beforeRequest: generateSendMessageParams
+          headers:
+            token: "{{ $processEnvironment.SWITCHBOARD_AUTH_TOKEN }}"
+          json:
+            query: |
+              mutation sendMessage($profileId: UUID!, $to: PhoneNumber!, $body: String!, $contactZipCode: ZipCode, $mediaUrls: [Url], $sendBefore: Datetime!) {
+                sendMessage(input: {profileId: $profileId, to: $to, body: $body, contactZipCode: $contactZipCode, mediaUrls: $mediaUrls, sendBefore: $sendBefore}) {
+                  outboundMessage {
+                    id
+                    createdAt
+                  }
+                }
+              }
+            variables:
+              profileId: "{{ $processEnvironment.SWITCHBOARD_PROFILE_ID }}"
+              to: "{{ toNumber }}"
+              body: "An SMS from Artillery load test!"
```

Comment:
Suggestion 😉 
```suggestion
              body: "An SMS from Artillery load test 💣!"
```

Reply:
Suggestion: Define this as a variable (if that's possible). Makes it easier if we expand to MMS tests etc. to copy this and not have to change it in multiple places